scss/_partners.scss

```diff
@@ -0,0 +1,40 @@
+@import 'config';
+
+.partners {
+  margin-top: 2rem;
+  background-color: $primary-color;
+  display: flex;
+  flex-direction: column;
+  color: $light-color;
+  padding: 1.5rem;
```

Comment:
- [ ] [REQUIRED] For the mobile version home page, we only have the more button, and then the footer is on the about page but your design has the more button and the footer on the home page. Kindly fix this.
### Your Design
![Screenshot from 2022-08-03 23-05-49](https://user-images.githubusercontent.com/58692590/182721472-a788d875-6922-401f-bd09-fb1ee902456c.png)

### Template provided
![Screenshot from 2022-08-03 23-13-04](https://user-images.githubusercontent.com/58692590/182721599-dd240b77-c707-4664-b469-5e974eb58475.png)

Reply:
mobile version of the page changed and about us page removed from this section to suit the template.problem solved